grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1278,6 +1278,18 @@ class QueryService implements Status {
         results
     }
 
+    @SuppressWarnings("GroovyAssignabilityCheck")
+    StorageDocument findAttachmentByUuid(String uuid) {
+        final session = sessionFactory.currentSession
+        final String query = ' select * from storage_document where uuid = :uniqueId '
```

Comment:
This will also find deleted documents

Reply:
That's true. I've modified this query like this ->
 
`SELECT * FROM storage_document WHERE uuid = :uuid AND deleted = 0`